frontend/src/components/app/app.tsx

```diff
@@ -31,6 +32,7 @@ const App: FC = () => {
           path={AppRoute.ROOT}
           element={<AuthorizedRoute component={<Dashboard />} />}
         />
+        <Route path={AppRoute.LOG_OUT} element={<LogOut />} />
```

Comment:
```suggestion
```
actually you don't need this route at all, you just need to call an action and remove user from the store